frontend/src/metabase/visualizations/visualizations/LinkViz/LinkViz.styled.tsx

```diff
@@ -55,7 +61,10 @@ export const SearchResultsContainer = styled.div`
   pointer-events: all;
 `;
 
-export const BrandIconWithHorizontalMargin = styled(Icon)`
-  color: ${color("brand")};
-  margin: 0 0.5rem;
+export const SearchResultsContainer = styled.div`
+  ${searchResultsStyles}
+`;
+
+export const StyledRecentsList = styled(RecentsList)`
+  ${searchResultsStyles}
 `;
```

Comment:
not sure if there's a better way to keep these styles DRY with emotion other than just assigning them to a string?